kartograf/collectors/parse.py

```diff
@@ -9,8 +11,8 @@
 
 @timed
 def parse_routeviews_pfx2as(context):
-    raw_file = f'{context.out_dir_collectors}pfx2asn.txt'
-    clean_file = f'{context.out_dir_collectors}pfx2asn_clean.txt'
+    raw_file = Path(context.out_dir_collectors) / "pfx2asn.txt"
+    clean_file = Path(context.out_dir_collectors) / "pfx2asn_clean.txt"
 
     print("Cleaning " + raw_file)
```

Comment:
Got a failure here

```
Traceback (most recent call last):
  File "/Users/FJ/projects/python/kartograf/./run", line 11, in <module>
    main()
  File "/Users/FJ/projects/python/kartograf/kartograf/cli.py", line 94, in main
    Kartograf.map(args)
  File "/Users/FJ/projects/python/kartograf/kartograf/kartograf.py", line 88, in map
    parse_routeviews_pfx2as(context)
  File "/Users/FJ/projects/python/kartograf/kartograf/timed.py", line 10, in wrapper
    result = func(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/FJ/projects/python/kartograf/kartograf/collectors/parse.py", line 17, in parse_routeviews_pfx2as
    print("Cleaning " + raw_file)
          ~~~~~~~~~~~~^~~~~~~~~~
TypeError: can only concatenate str (not "PosixPath") to str
```

Probably just needs this:

```suggestion
    print("Cleaning " + str(raw_file))
```